jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/colibri/ColibriV2SessionManager.kt

```diff
@@ -250,11 +251,14 @@ class ColibriV2SessionManager(
             // The requests for each session need to be sent in order, but we don't want to hold the lock while
             // waiting for a response. I am not sure if processing responses is guaranteed to be in the order in which
             // the requests were sent.
-            val bridge = bridgeSelector.selectBridge(getBridges(), participant.region, bridgeVersion)
-                ?: run {
-                    eventEmitter.fireEvent { bridgeSelectionFailed() }
-                    throw BridgeSelectionFailedException()
-                }
+            val bridge = bridgeSelector.selectBridge(
+                getBridges(),
+                ParticipantProperties(participant.region),
+                bridgeVersion
```

Comment:
Should `bridgeVersion` go to `ConferenceBridgeProperties` now?

Reply:
It's not a bad idea but ConferenceBridgeProperties was the previous PR. :slightly_smiling_face:

So probably that should be a separate PR if we want to do it. I can look at doing it next week.